Makefile

```diff
@@ -1,10 +1,30 @@
 all: test
 
 # Packages to install if we are running on Debian
+PACKAGES := \
+    python3-django \
+    python3-flake8 \
+    python3-jinja2 \
+    python3-pytest-django \
+    python3-gunicorn \
+    python3-coverage \
+    python3-pytest-cov \
+    python3-django-redis \
+    python3-ipy \
+    python3-django-ratelimit \
+    python3-django-cors-headers \
+    python3-django-oauth-toolkit \
+
+# TODO - from the requirements.txt
+# pyserial>=3.2.1
+# RPi.GPIO>=0.6.3
+# paho-mqtt>=1.2
```

Comment:
mqtt could be dropped completely, it was never put in use anyway

Reply:
yeah, I thought I remembered that